src/Analysis/AtlasDataDictionary/Components/ColumnHeaders/ColumnHeaders.tsx

```diff
@@ -1,125 +1,154 @@
+      show: columnManagementData.vocabularyID,
+      jsx: (
+        <span>
+          Vocabulary
+          <br />
+          ID
+        </span>
+      ),
+    },
+    {
+      headerKey: 'conceptID',
+      show: columnManagementData.conceptID,
+      jsx: (
+        <span>
+          Concept
+          <br />
+          ID
+        </span>
+      ),
+    },
+    {
+      headerKey: 'conceptCode',
+      show: columnManagementData.conceptCode,
+      jsx: (
+        <span>
+          Concept
+          <br />
+          Code
+        </span>
+      ),
+    },
+    {
+      headerKey: 'conceptName',
+      show: columnManagementData.conceptName,
+      jsx: (
+        <span>
+          Concept
+          <br />
+          Name
+        </span>
+      ),
+    },
+    {
+      headerKey: 'conceptClassID',
+      show: columnManagementData.conceptClassID,
+      jsx: (
+        <span>
+          Concept
+          <br />
+          Class&nbsp;ID
+        </span>
+      ),
+    },
+    {
+      headerKey: 'numberOfPeopleWithVariable',
+      show: columnManagementData.numberPercentPeopleWithVariable,
+      jsx: (
+        <span>
+          #&nbsp;/&nbsp;%&nbsp;of&nbsp;People
+          <br />
+          with&nbsp;Variable
+        </span>
+      ),
+    },
+    {
+      headerKey: 'numberOfPeopleWhereValueIsFilled',
+      show: columnManagementData.numberPercentOfPeopleWhereValueIsFilled,
+      jsx: (
+        <span>
+          #&nbsp;/&nbsp;%&nbsp;of&nbsp;People
+          <br />
+          where&nbsp;Value&nbsp;is&nbsp;Filled
+        </span>
+      ),
+    },
+    {
+      headerKey: 'numberOfPeopleWhereValueIsNull',
+      show: columnManagementData.numberPercentOfPeopleWhereValueIsNull,
+      jsx: (
+        <span>
+          #&nbsp;/&nbsp;%&nbsp;of&nbsp;People
+          <br />
+          where&nbsp;Value&nbsp;is&nbsp;Null
+        </span>
+      ),
+    },
+    {
+      headerKey: 'valueStoredAs',
+      show: columnManagementData.valueStoredAs,
+      jsx: (
+        <span>
+          Value
+          <br />
+          Stored&nbsp;As
+        </span>
+      ),
+    },
+    {
+      headerKey: 'valueSummary',
+      show: columnManagementData.valueSummary,
+      jsx: <span>Value&nbsp;Summary</span>,
+    },
+  ];
 
-const ColumnHeaders = ({ handleSort, sortConfig }: ITableHeadersProps) => (
-  <thead className={'column-headers'} data-testid='column-headers'>
-    <tr>
-      {headerItems.map((item) => (
-        <Header
-          key={item.headerKey}
-          handleSort={handleSort}
-          headerJSX={item.jsx}
-          headerKey={item.headerKey}
-          sortConfig={sortConfig}
-          sortable={item.headerKey !== 'valueSummary' && item.headerKey !== ''}
-        />
-      ))}
-    </tr>
-  </thead>
-);
+  return (
+    <thead className={'column-headers'} data-testid='column-headers'>
+      <tr>
+        {headerItems.map((item) => (
+          <React.Fragment key={item.headerKey}>
+            {item.show && (
```

Comment:
maybe this could also be written as follows?
```suggestion
            {item.columnManagementData[item.headerKey] && (
```
Or am I overlooking something? Just wondering why this new `show` attribute was needed...

Reply:
This is a very good point. 
To prevent or enable and thus hide / show the columns this has been updated to: 
```          {columnManagementData[item.headerKey] && ( ```

The column management headerKeys have been updated to use the same naming convention as what is being used elsewhere in the application, such as for sorting.